.changelog/14140.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli: `acl policy info` output format has changed to improve readability with large policy documents
```

Comment:
This is the only bit of this that's changing already-shipped code. If we want to be fussy we can pull this out to its own PR but it's a tiny change that won't get backported so seemed safe to keep here.